src/stories/elements/controlledDisplayFiltersGroup/ControlledDisplayFiltersGroup.stories.tsx

```diff
@@ -16,9 +16,52 @@ const Template: StoryFn<typeof ControlledDisplayFiltersGroup> = (args) => (
   <ControlledDisplayFiltersGroup {...args} />
 );
 
-export const ControlledDisplayFilterBasic: StoryFn<typeof ControlledDisplayFiltersGroup> =
+export const displayFilterBasic: StoryFn<typeof ControlledDisplayFiltersGroup> = Template.bind({});
+displayFilterBasic.args = {
+  filters: [
+    {
+      label: 'metadata_key_1',
+      onChange: (value) => console.log('changed', value)
+    },
+    {
+      label: 'metadata_key_2',
+      onChange: (value) => console.log('changed', value)
+    },
+    {
+      label: 'metadata_key_3',
+      onChange: (value) => console.log('changed', value)
+    }
+  ],
+  toggleShowAll: (value) => console.log('toggle show all', value),
+  toggleAllLabel: 'Show all'
+};
+
+export const ControlledDisplayFilterBasic1: StoryFn<typeof ControlledDisplayFiltersGroup> =
```

Comment:
Why 3 `ControlledDisplayFilterBasic`s? I don't think I get the difference between them.